src/higher-level/param-hl.c

```diff
@@ -591,6 +591,54 @@ int *band_ptr[_WVL_LENGTH_] = {
         v[_WVL_NIR_] = v[_WVL_SWIR1_] = true;
         copy_string(tsa->index_name[idx], NPOW_02, "NDM"); 
         break;
+      case _IDX_KNV_:
```

Comment:
I also added the case for KNDVI here - it was missing for some reason. I think that's why I did not add anything here before, because I searched the whole repository for 'KNV' and 'KNDVI'...

Reply:
Ah, yes, there was something missing in master. The addition was already in develop. I am also changing the base for the PR to develop, and resolve the merge conflict now.